apps/redi-connect/tailwind.config.js

```diff
@@ -0,0 +1,13 @@
+const { createGlobPatternsForDependencies } = require('@nrwl/react/tailwind')
+const { join } = require('path')
+
+module.exports = {
+  content: [
+    join(__dirname, './src/**/*!(*.stories|*.spec).{js,ts,jsx,tsx,html}'),
+    ...createGlobPatternsForDependencies(__dirname),
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
```

Comment:
@coderabbitai can you explain how this works? Especially the createGlobPatternsForDependencies part.